streamx-flink/streamx-flink-shims/streamx-flink-shims-base/src/main/scala/com/streamxhub/streamx/flink/core/FlinkSqlExecutor.scala

```diff
@@ -170,11 +173,12 @@ object FlinkSqlExecutor extends Logger {
         case _ =>
       }
     } else {
-      logError("No 'INSERT' statement to trigger the execution of the Flink job.")
-      throw new RuntimeException("No 'INSERT' statement to trigger the execution of the Flink job.")
+      logError("An 'INSERT' statement is required to trigger the execution of the FLink task.")
```

Comment:
The prompt here should be clearer and more detailed, and directly tell the user the error, rather than the conditions that the user needs to meet

----

这里提示应该更加清晰明细一些,直接告诉用户错误,而不是告诉用户需要满足的条件

Reply:
OK